cmd/loadtest/loadtest.go

```diff
@@ -34,7 +34,6 @@ import (
 	ethereum "github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	ethcommon "github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core/types"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
```

Comment:
```suggestion
	"github.com/ethereum/go-ethereum/core/types"
```

Reply:
> 

For this one I already removed `types.*` occurences in favour of `ethtypes.*`.